content/warnings/refs-must-have-owner.md

```diff
@@ -54,10 +56,13 @@ ReactDOM.render(
 );
 ```
 
-Consider if you [really need a ref](/docs/refs-and-the-dom.html#when-to-use-refs) before using this approach.
+Zanim zdecydujesz się na takie rozwiązanie, upewnij się, [czy na pewno potrzebujesz referencji](/docs/refs-and-the-dom.html#when-to-use-refs).
 
-## Multiple copies of React {#multiple-copies-of-react}
+## Wiele kopii Reacta {#multiple-copies-of-react}
 
-Bower does a good job of deduplicating dependencies, but npm does not. If you aren't doing anything (fancy) with refs, there is a good chance that the problem is not with your refs, but rather an issue with having multiple copies of React loaded into your project. Sometimes, when you pull in a third-party module via npm, you will get a duplicate copy of the dependency library, and this can create problems.
+Bower dobrze radzi sobie z usuwaniem zduplikowanych zależności, ale npm już nie.
+Jeśli nie robisz nic (szczególnego) z referencjami, jest spora szansa, że problem nie leży postronie Twoich referencji,
```

Comment:
```suggestion
Jeśli nie robisz nic (szczególnego) z referencjami, jest spora szansa, że problem nie leży postronie twoich referencji,
```
W dokumentacji zaimki piszemy małymi literami.

Reply:
To jest dyskusyjne: https://sjp.pwn.pl/poradnia/haslo/Twoj-czy-twoj;2426.html
O ile nigdzie nie piszemy z wielkiej, to luz.

Reply:
Dokładnie - nigdzie nie piszemy wielką literą, stąd moja uwaga :-)